django/contrib/admin/templates/admin/index.html

```diff
@@ -17,7 +17,7 @@
         <div class="app-{{ app.app_label }} module">
         <table>
         <caption>
-            <a href="{{ app.app_url }}" class="section" title="{% blocktrans with name=app.name %}Models in the {{ name }} application{% endblocktrans %}">{{ app.name }}</a>
+            <a href="{{ app.app_url }}" class="section" title="{% blocktrans with name=app.name force_escape %}Models in the {{ name }} application{% endblocktrans %}">{{ app.name }}</a>
```

Comment:
Change to 2 spaces for indentation https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style